config.rb

```diff
@@ -95,6 +95,8 @@ def self.test?
   optional :github_app_webhook_secret, string, clear: true
   optional :vm_pool_project_id, string
   optional :github_runner_service_project_id, string
+  optional :github_test_installation_id, string
+  optional :github_test_access_token, string
```

Comment:
We also have `ci_hetzner_sacrificial_server_id` for E2E tests, denoted by the `ci` prefix. As the required configurations for E2E tests are growing, it's becoming difficult to distinguish between those for E2E and those for production. Could we relocate all three configurations to distinct section with a prefix for clarity such as:

```
# E2E Tests
e2e_hetzner_server_id
e2e_github_installation_id
e2e_github_access_token
```

Reply:
I've moved github related one to E2E tests part in the config, I'll do it for sacrificial server with subsequent PR